src/ring_theory/localization.lean

```diff
@@ -42,14 +46,21 @@ structure with an isomorphic one; one way around this is to isolate a predicate
 a structure up to isomorphism, and reason about things that satisfy the predicate.
 
 A ring localization map is defined to be a localization map of the underlying `comm_monoid` (a
-`submonoid.localization_map`) which is also a ring hom. To prove most lemmas about a
+`submonoid.localization`) which is also a ring hom. To prove most lemmas about a
```

Comment:
Are these changes correct?

Reply:
oh no! Thank you; changing them back now.